src/data/community-meetups.json

```diff
@@ -89,6 +89,12 @@
     "location": "Jakarta",
     "link": "https://www.meetup.com/Ethereum-indonesia/"
   },
+  {
+    "title": "Ethereum Jakarta - Jakarta-focused Eth User Group",
```

Comment:
```suggestion
    "title": "Ethereum Jakarta",
```

Reply:
We only want to use the title of the event, we don't need to include a description here.